package.json

```diff
@@ -1,25 +1,26 @@
 {
-  "name": "hetmech-frontend",
-  "version": "0.1.0",
-  "private": true,
-  "dependencies": {
-    "react": "^16.7.0",
-    "react-dom": "^16.7.0",
-    "react-scripts": "2.1.3"
-  },
-  "scripts": {
```

Comment:
I actually like the `scripts` section in react app. It wraps all webpack config and you can use "npm run test" for testing or "npm run build" for building. Did you try them?

Reply:
I did try them. But they come with [many dependencies](https://www.npmjs.com/package/react-scripts?activeTab=dependencies) that are not necessary for our purposes. In addition, if you ever need to modify the webpack config (which in my experience will be highly likely), you have to run `eject` on react scripts, which exposes the webpack config and forces you to maintain it from that point on anyway.

Reply:
If you feel comfortable to work with webpack directly, then please feel free to keep it. No big deal.